drivers/hmis/lib/hmis_util/json_forms.rb

```diff
@@ -179,33 +222,98 @@ def load_definition(form_definition:, identifier:, role:)
         role: role,
         version: 0,
         status: 'draft',
-      ).first_or_create!
+      ).first_or_create!(title: title || role.to_s.humanize)
       record.definition = form_definition
+      record.title = title if title.present?
       record.save!
     end
 
-    # Load form definitions for editing and creating records
-    public def seed_record_form_definitions
-      record_forms.merge(service_forms).each do |identifier, form_definition|
+    public def ensure_system_instances_exist!
+      [
+        :project,
+        :organization,
+        :project_coc,
+        :funder,
+        :inventory,
+        :client,
+        :new_client_enrollment,
+        :enrollment,
+      ].each do |identifier|
         role = identifier.upcase.to_sym
-        role = :SERVICE if service_forms.key?(identifier)
-        raise "Unrecognized record form: #{identifier}" unless Hmis::Form::Definition::FORM_ROLES.key?(role)
+        raise "Unrecognized record form: #{identifier}" unless Hmis::Form::Definition::SYSTEM_FORM_ROLES.include?(role)
 
-        load_definition(
-          form_definition: form_definition,
-          identifier: identifier,
-          role: role,
-        )
+        default_instance = Hmis::Form::Instance.defaults.where(definition_identifier: identifier).first_or_create!
+        default_instance.update(system: true, active: true)
+        default_instance.touch
+      end
+    end
+
+    # Create some default HUD occurrence point instances
+    public def create_default_occurrence_point_instances!
+      # Move-in Date
+      unless Hmis::Form::Instance.where(definition_identifier: 'move_in_date').exists?
+        [3, 9, 10, 13].each do |ptype|
```

Comment:
Can we pull this list from HudUtility2024?

Reply:
I can add it but I didn't want to conflict with your 2024 branch where you also added in project groups to Hud Utility. Since yours are used by the warehouse, I figured I could update the HMIS ones once your branch goes in

Reply:
Updated - how does that look?